ios/RNNativePlayground/ViewController.m

```diff
@@ -108,9 +108,8 @@ - (NSString *)localizedStringWithKey:(NSString *)key {
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch {
   
   if (self.navigationController.interactivePopGestureRecognizer == gestureRecognizer) {
-    return [_vc isInteractivePopGestureAllowed];
+    return [_activeVc isInteractivePopGestureAllowed];
```

Comment:
Was this change necessary?

Reply:
Yes. Before there was only `vc`. Now there are `activeVc` and `hotelsVc`, but we would like to disable gestures in the currently opened view controller.